chemprop/args.py

```diff
@@ -796,13 +1024,15 @@ def process_args(self) -> None:
         super(PredictArgs, self).process_args()
 
         if self.regression_calibrator_metric is None:
-            if self.calibration_method == 'zelikman_interval':
-                self.regression_calibrator_metric = 'interval'
+            if self.calibration_method == "zelikman_interval":
+                self.regression_calibrator_metric = "interval"
```

Comment:
This block makes "interval" the default output type if zelikman_interval is used for calibration. Would you want to make it the default for conformal regression too? Or are you making a whole new output type, in which case maybe it should be added as an option for regression_calibrator_metric.

Reply:
The conformal method doesn't use the regression_calibrator_metric.